code/modules/admin/view_variables/filterrific.dm

```diff
@@ -78,7 +78,8 @@
 				target.update_filters()
 				. = TRUE
 		if("mass_apply")
-			if(!check_rights_for(usr.client, R_DEBUG))
+			if(!check_rights_for(usr.client, R_EVENT))
+				to_chat(usr, SPAN_USERDANGER("Stay in your lane, jannie."))
```

Comment:
message aside, is the change from R_DEBUG to R_EVENT intended ? might need to bring this up with server management

Reply:
nope, it (with the message) was just upstream. both nuked